cmd/mountlib/mount.go

```diff
@@ -149,6 +149,9 @@ Stopping the mount manually:
     # OS X
     umount /path/to/local/mount
 
+**Note**: As of `rclone` 1.52.2, `rclone mount` now requires Go version 1.13 or newer due to
```

Comment:
Note that you can't use ` inside a ` string. You'll have to remove them or use " as appropriate.

Reply:
Fixed.